Tests/ARTFallbackTest.m

```diff
@@ -44,7 +44,11 @@ - (void)testAllHostsIncludedOnce {
 }
 
 - (void)testCustomFallbackHosts {
-    NSArray *customHosts = @[@"test.ably.com", @"test2.ably.com", @"test3.ably.com"];
+    NSArray *customHosts = @[@"test.ably.com",
+                             @"test2.ably.com",
+                             @"test3.ably.com",
+                             @"test4.ably.com",
+                             @"test5.ably.com"];
```

Comment:
5x4x3x2 = 1 in 120 test will fail.
Why not just go for 15 hosts, for example, giving you a 1 in 479001600 change the test will fail?